lighthouse-core/report/html/report-styles.css

```diff
@@ -1158,18 +1164,18 @@
 .lh-table {
   --image-preview-size: 24px;
   border-collapse: collapse;
+  /* Can't assign padding to table, so shorten the width instead. */
+  width: calc(100% - var(--inner-audit-left-padding));
 }
 
-.lh-table thead {
-  background-color: var(--lh-table-higlight-bg);
-}
 .lh-table thead th {
```

Comment:
we had something in there about text alignment with headers


![image](https://user-images.githubusercontent.com/39191/57164397-d4955300-6da8-11e9-989b-a0f120165890.png)

1. getting description flush left with the title.
1. getting URL flush left with those two.

right, @yuinchien?

Reply:
hm i aligned it to the description instead

Reply:
@Hoten the title, description, the table heading(URL), the table should all align left to the same line. 

<img width="452" alt="Screen Shot 2019-05-03 at 1 47 23 PM" src="https://user-images.githubusercontent.com/406636/57164782-0955da00-6daa-11e9-9676-380b0959f903.png">

Reply:
@Hoten can we horizontally center the shape (triangle) with the first line of title? Thanks!
<img width="112" alt="Screen Shot 2019-05-03 at 1 48 48 PM" src="https://user-images.githubusercontent.com/406636/57164846-3609f180-6daa-11e9-9a29-3b927ce6bd02.png">

Reply:
done